src/test/java/net/masterthought/cucumber/json/EmbeddingTest.java

```diff
@@ -68,7 +68,7 @@ public void getFileName_ReturnsFileName() {
     public void getFileName_ReturnsFileNameForSVG() {
 
         // given
-        Embedding embedding = new Embedding("image/svg+xml", "some data");
+        Embedding embedding = new Embedding("image/svg+xml", "some data", null);
```

Comment:
then consider to leave old constructor and add new one below

Reply:
I updated the PR with two constructors for Embedding.